common/core/overlaylayer.cpp

```diff
@@ -263,6 +255,15 @@ void OverlayLayer::InitializeState(HwcLayer* layer,
     }
   }
 
+ // In case of layer using blening we need to force partial clear. Otherwise
```

Comment:
Why does moving the code chunk here fix the issues ?

Reply:
swap should be  done on layer-> GetLayerDamage as we are rotating display to 90.
But in this case the swap was done on layer->GetDisplayFrame  which caused incomplete screen.

Reply:
hmm but u still use DisplayFrame right ?

Reply:
Yes we use DisplayFrame after swapping the surface damage co-ordinates.

Reply:
After the rotation is done, the source layer dimensions are already updated, so why do we need to do swap again?